pkg/transfers/inbound/cleanup_test.go

```diff
@@ -43,3 +72,149 @@ func TestCleanupErr(t *testing.T) {
 		t.Errorf("unexpected deleted file: %s", agent.DeletedFile)
 	}
 }
+
+func Test_CleanupEmptyFiles_InboundPath_Success(t *testing.T) {
+	agent := &upload.MockAgent{}
+
+	dir, _ := ioutil.TempDir("", "clenaup-testing")
+	dl := &downloadedFiles{dir: dir}
+	defer dl.deleteFiles()
+
+	// write a test file to attempt deletion
+	path := filepath.Join(dl.dir, agent.InboundPath())
+	if err := os.MkdirAll(path, 0777); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := ioutil.WriteFile(filepath.Join(path, "empty_file.ach"), []byte(""), 0644); err != nil {
+		t.Fatal(err)
+	}
+
+	files, err := ioutil.ReadDir(path)
+	if err != nil {
+		t.Fatal(err)
+	}
+	fileInfo := files[0]
+	mockTimeNow := fileInfo.ModTime().Add(10 * time.Minute)
+	minutesAfterToDelete := 5 * time.Minute
+
+	if err := CleanupEmptyFiles(log.NewNopLogger(), agent, dl, mockTimeNow, minutesAfterToDelete); err == nil {
+		t.Error("expected error")
+	}
+
+	if agent.DeletedFile != "inbound/empty_file.ach" {
+		t.Errorf("unexpected deleted file: %s", agent.DeletedFile)
+	}
+}
+
+func Test_CleanupEmptyFiles_ReturnPath_Success(t *testing.T) {
+	agent := &upload.MockAgent{}
+
+	dir, _ := ioutil.TempDir("", "clenaup-testing")
+	dl := &downloadedFiles{dir: dir}
+	defer dl.deleteFiles()
+
+	// write a test file to attempt deletion
+	path := filepath.Join(dl.dir, agent.ReturnPath())
+	if err := os.MkdirAll(path, 0777); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := ioutil.WriteFile(filepath.Join(path, "empty_file.ach"), []byte(""), 0644); err != nil {
+		t.Fatal(err)
+	}
+
+	files, err := ioutil.ReadDir(path)
+	if err != nil {
+		t.Fatal(err)
+	}
+	fileInfo := files[0]
+	mockTimeNow := fileInfo.ModTime().Add(10 * time.Minute)
+	minutesAfterToDelete := 5 * time.Minute
+
+	if err := CleanupEmptyFiles(log.NewNopLogger(), agent, dl, mockTimeNow, minutesAfterToDelete); err == nil {
+		t.Error("expected error")
+	}
+
+	if agent.DeletedFile != "return/empty_file.ach" {
+		t.Errorf("unexpected deleted file: %s", agent.DeletedFile)
+	}
+}
+
+func Test_CleanupEmptyFiles_Not_Run_When_Config_Is_Zero(t *testing.T) {
```

Comment:
I think these might be easier to read with fewer underscores.. Maybe something like `TestCleanupEmptyFiles_notRunWhenConfigZero(t *testing.T)`?

Reply:
Agreed. I can clean these up here shortly.